### PR TITLE
fix(plugin-server-split): only load scheduled tasks in servers that process them

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Problem

We're unnecessarily loading schedules on servers that won't ever use them

## Changes

Check capabilities for loading schedule

## How did you test this code?

Added a test

### CODE REVIEW

File: plugin-server/src/worker/plugins/loadPluginsFromDB.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { PluginAttachment } from '@posthog/plugin-scaffold'
```
Comment (yakkomajuri) [inline plugin-server/src/worker/plugins/loadPluginsFromDB.ts:1]: just moved

File: plugin-server/src/worker/plugins/setup.ts [hunk 2]
```diff
@@ -41,107 +39,18 @@ export async function setupPlugins(server: Hub): Promise<void> {
...
-            }
-        }
+    for (const teamId of hub.pluginConfigsPerTeam.keys()) {
+        hub.pluginConfigsPerTeam.get(teamId)?.sort((a, b) => a.order - b.order)
     }
 
-    if (count > 0) {
-        status.info('🔌', `Loaded ${count} scheduled tasks`)
+    // Only load the schedule in server that can process scheduled tasks, else the schedule won't be useful
+    if (hub.capabilities.pluginScheduledTasks) {
```
Comment (yakkomajuri) [inline plugin-server/src/worker/plugins/setup.ts:53]: the one actual change :D

**Review by hazzadous — APPROVED** [review]
LGTM although there are some tests failing?

**Comment by hazzadous** [comment]
🥳

**Pull request merged**